python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -874,7 +889,23 @@ def __name__(self, value):
         setattr(self._fsproxy_slow, "__name__", value)
 
 
-def _fast_slow_function_call(func: Callable, /, *args, **kwargs) -> Any:
+def _assert_fast_slow_eq(left, right, **kwargs):
+    assert_func = (
+        assert_eq
+        if not kwargs.get("assert_func")
+        else kwargs.get("assert_func")
+    )
```

Comment:
`get` is designed to handle exactly this case.
```suggestion
    assert_func = kwargs.get("assert_func", assert_eq)
```

However, I'm not sure that this is the right approach, see below.